packages/atomic/cypress/fixtures/test-fixture.ts

```diff
@@ -0,0 +1,129 @@
+type SearchInterface = HTMLElement & {
+  initialize: (opts: {
+    accessToken: string;
+    organizationId: string;
+  }) => Promise<void>;
+  executeFirstSearch: () => void;
+};
+
+export type TestFeature = (e: TestFixture) => void | Promise<void>;
+
+export type TagProps = Record<string, string>;
+
+export class TestFixture {
+  private urlParams: {key: string; value: string}[] = [];
+  private aliases: TestFeature[] = [];
+  private testURL = 'http://localhost:3333/pages/test.html';
+  private execFirstSearch = true;
+
+  constructor() {
+    cy.visit(this.testURL).injectAxe();
+    this.intercept();
+
+    cy.document().then((doc) => {
```

Comment:
I would rather inject all the html inside the page at the same time, not to run in similar issues that we did before:
https://github.com/coveo/ui-kit/pull/668/files

Reply:
I could modify the init function to inject everything at once, yes, but if the previous problem was executing the initial search too quick, this should be fixed/should not matter with the way it's done now 🤔